### PR TITLE
data:krimoelbahri

### PR DESCRIPTION
**Pull request by krimoelbahri**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7374"></a>

### CODE REVIEW

File: data/krimoelbahri.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Karim Guezlane",
+  "bio": "Math teacher diving into web developement",
+  "type":"personal",
```
Comment (krishstwt) [inline data/krimoelbahri.json:4]: The type field is no longer needed, the rest is awesome, thank you!
```suggestion
```

**Review by krishstwt — DISMISSED** [review]

**Comment by krimoelbahri** [comment]
@krishstwt  
I changed the file as you suggested

**Review by krishstwt — APPROVED** [review]
Looks great!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**